psi4/src/export_mints.cc

```diff
@@ -68,7 +68,6 @@
 #include "psi4/libmints/dipole.h"
```

Comment:
I dont know if you clang-format is picking up our limited style sheet as there shouldn't be this many changes in this file. These options are:
```
BasedOnStyle: Google
IndentWidth: 4
ColumnLimit: 120
SortIncludes: false
```
and can be found in `psi4/.clang-format`. It looks like you are truncating at 90 line-width, not 120.

Reply:
I am using the standard command: `clang-format -style=file -i export_mints.cc`. This should automatically use the `psi4/.clang-format`file for formatting.